extractor/gradle/libs.versions.toml

```diff
@@ -0,0 +1,10 @@
+# This file was generated by the Gradle 'init' task.
+# https://docs.gradle.org/current/userguide/platforms.html#sub::toml-dependencies-format
+
+[versions]
+guava = "32.1.3-jre"
+junit = "4.13.2"
```

Comment:
This section should exist only if version is references twice .
Please inline version values